_includes/graphql/users.md

```diff
@@ -272,9 +274,84 @@ mutation resetPassword {
 }
 ```
 
+The emailed password reset link will GET the Parse REST API to verify the token is still valid.  For example:
+```
+https://www.example.com/parse/apps/APP_ID/request_password_reset?token=xxxxxxxxxxxx&username=test%40example.com
+```
```

Comment:
Thanks for the PR! Maybe also explain how to set www.example.com in Parse Server?

Reply:
Hi @daviddoran the "example.com" in that URL comes from the user's email when they register for an account.

Reply:
Ok. Makes sense. I hadn't fully understood the example.